src/Microsoft.DiaSymReader/SymUnmanagedFactory.cs

```diff
@@ -79,18 +78,13 @@ internal static string DiaSymReaderModuleName
 
             return null;
         });
-#endif
 
         // internal for testing
         internal static string GetEnvironmentVariable(string name)
         {
             try
             {
-#if NET20
-                return Environment.GetEnvironmentVariable(name);
```

Comment:
>Environment.GetEnvironmentVariable(name); [](http://example.com/codeflow?start=23&length=41)

This was a workaround for netstandard1.1, the NET20 code is certainly better :)

Reply:
Ah okay. Will fix that.